cardano-node/src/Cardano/Node/TUI/GitRev.hs

```diff
@@ -13,7 +13,7 @@
 
 {-# OPTIONS_GHC -Wno-simplifiable-class-constraints #-}
 
-module Cardano.Node.GitRev (
+module Cardano.Node.TUI.GitRev (
```

Comment:
What does TUI stand for?

Reply:
https://en.wikipedia.org/wiki/Text-based_user_interface

Reply:
Ah, okay. I wonder if `TerminalUI` or `TextUI` might be more clear. Unless `TUI` is pretty common and people generally understand it (@deepfire, what do you think?). Perhaps it's just me who hasn't seen this acronym before.

Reply:
Either way, I wouldn't block the PR on this

Reply:
I think `TUI` is good.